includes/api/save-abandonment-data.php

```diff
@@ -122,6 +122,16 @@ function saveCartAbandonmentData(WP_REST_Request $request)
     return new WP_REST_Response($response, $statusCode);
 }
 
+
+//Check CartBounty plugin is activated or not 
+if (is_plugin_active('woo-save-abandoned-carts/cartbounty-abandoned-carts.php') && empty($customerEmail) == false) {
+
```

Comment:
@angbpy Remove extra white spaces. Maintain the same code format.

Reply:
@ChetanGN Done. Please check.